tests/users/test_user_register.py

```diff
@@ -0,0 +1,167 @@
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+
+class UserRegisterTestCase(APITestCase):
+    def test_success_with_correct_credentials(self):
+        data={
+            "first_name": "string1",
+            "last_name": "string2",
+            "email": "user234@example.com",
+            "contact": "string9000",
+            "password": "string989"
+        }
+
+        response = self.client.post("/users/register/", data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_fail_with_used_email(self):
+        data = {
+            "first_name": "string1",
+            "last_name": "string2",
+            "email": "user234@example.com",
+            "contact": "string9000",
+            "password": "string989"
+        }
+
+        response = self.client.post("/users/register/", data)
+        response = self.client.post("/users/register/", data)

```

Comment:
```suggestion
```

Reply:
@anshsrtv sir this is used first time so as to register it as a new user and next time it is used so as to generate 400 bad request error for already registered or used email.

Reply:
Okay cool 😆

Reply:
> Nicely done 💯 Just make the suggested changes and delete `users/tasks.py` and `users/forms.py` if they aren't being used anywhere.

@anshsrtv sir, 
I don't find the usage of tasks.py anywhere so i will delete it but forms.py is used in admin.py so please can you tell me whether to add or delete.